src/main.rs

```diff
@@ -1,10 +1,39 @@
+extern crate egg_mode;
+extern crate dotenv;
 extern crate fs2;
 extern crate rand;
+extern crate tokio_core;
 
 mod name_gen;
 mod epic;
 
+use dotenv::dotenv;
+use std::env;
```

Comment:
It's nice to sort `use` so that `std` come first, then a blank line, then all external libraries, then a blank line, then all internal modules.